.github/workflows/linter.yml

```diff
@@ -0,0 +1,32 @@
+# This workflow executes several linters on changed files based on languages used in your code base whenever
```

Comment:
Why are you doing this with GH actions and not GitLab?

Reply:
GH actions was easier for me.
I will migrate to GL at a later stage.